main_chain.js

```diff
@@ -916,6 +916,7 @@ function markMcIndexStable(conn, mci, onDone){
 function getSimilarMcis(mci){
```

Comment:
@dindinw what do you think of rewriting the whole function as
```
 function getSimilarMcis(mci){
  	var arrSimilarMcis = [];
  	if (mci <= 0)
  	  	return arrSimilarMcis;
  	var divisor = 10;
  	while (mci % divisor === 0) {
  		arrSimilarMcis.push(mci - divisor);
 		divisor *= 10;
 	}
 	return arrSimilarMcis;
 }
```

Reply:
I think your code is better 👍